packages/@aws-cdk/aws-ecs/test/test.scheduled-ecs-task.ts

```diff
@@ -0,0 +1,47 @@
+import { expect, haveResource } from '@aws-cdk/assert';
+import ec2 = require('@aws-cdk/aws-ec2');
+import cdk = require('@aws-cdk/cdk');
+import { Test } from 'nodeunit';
+import ecs = require('../lib');
+import { ScheduledEc2Task } from '../lib';
+
+export = {
+  "Can create a scheduled Ec2 Task without a service"(test: Test) {
```

Comment:
The phrasing of this unit test is a bit confusing. Is this just saying that we expect this construct to be a wrapper for an ECS task rather than a service? Also are there other API on the construct we should be unit testing?

Reply:
Is this just saying that we expect this construct to be a wrapper for an ECS task rather than a service? - Yes
Also are there other API on the construct we should be unit testing? - Fair point, adding more unit tests.